compiler/cpp/test/ReaderTest.cpp

```diff
@@ -0,0 +1,234 @@
+//
+// Created by Marc on 27.05.2021.
+//
+
+#include <reader/Reader.h>
+#include <gtest/gtest.h>
+
+// -------------------------------------------------------- EOF --------------------------------------------------------
+
+TEST(ReaderTests, EoF) {
```

Comment:
Can you use actual strings for the second argument? Looks a bit odd like that...

Reply:
Unfortunately, this is not possible. You can read more details about a feature request regarding this [here](https://github.com/google/googletest/issues/1594).